.travis.yml

```diff
@@ -15,7 +15,7 @@ _install: &_install
   - gimme 1.8
   - source ~/.gimme/envs/latest.env
   - pip install --upgrade pip
-  - pip install tensorflow
+  - pip install tensorflow keras
   - pip install -r requirements.txt codecov
```

Comment:
We decided to put Keras into the requirements because it does not depend on Tensorflow directly. Thus this is not needed